lib/src/main/java/org/team100/lib/motor/Neo550CANSparkMotor.java

```diff
@@ -0,0 +1,36 @@
+package org.team100.lib.motor;
+
+import org.team100.lib.config.Feedforward100;
+import org.team100.lib.config.PIDConstants;
+import org.team100.lib.telemetry.SupplierLogger;
+
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+
+/**
+ * Neo550 motor.
+ * 
+ * @see https://www.revrobotics.com/rev-21-1651/
+ */
+public class Neo550CANSparkMotor extends CANSparkMotor {
+    public Neo550CANSparkMotor(
+            SupplierLogger parent,
+            int canId,
+            MotorPhase motorPhase,
+            int currentLimit,
+            Feedforward100 ff,
+            PIDConstants pid) {
+        super(parent, new CANSparkMax(canId, MotorType.kBrushless),
+                motorPhase, currentLimit, ff, pid);
+    }
+
+    @Override
+    public double kROhms() {
+        return 0.12;
+    }
+
+    @Override
+    public double kTNm_amp() {
+        return 0.3944;
```

Comment:
where did this number come from?  0.39 Nm per amp is an enormous number, it means at the stall current of 100 amps the motor will be making 39 Nm which is like the tightening torque on a 5/16" bolt, i.e. huge compared to the output of a neo 550.

Reply:
I might need some help on this, how I did it was I took the torque NM from the graph and devided it by the amps.